modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,20 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+Argo CD cannot map to users if they have a direct `ClusterRolfeBinding` role. You can manually change users' role as `role:admin` on SSO through OpenShift.
```

Comment:
I would eliminate `A quick fix is to:` and instead, include `Procedure` before the steps (like you've done in other procedure modules). I think the intro you have here lets the users know that these steps are intended to change the users' role.